src/main/webapp/dev/DatePickerJsTest.js

```diff
@@ -0,0 +1,22 @@
+QUnit.module('datepicker.js');
+
+QUnit.test('getMinDateForPublishDate(visibleDate)', function(assert) {
+    assert.deepEqual(getMinDateForPublishDate(new Date('10/13/2014')), new Date('10/13/2014'), 'returns visibleDate');
+});
+
+QUnit.test('getMaxDateForVisibleDate(startDate,publishDate)', function(assert) {
```

Comment:
Should be `getMaxDateForVisibleDate(startDate, publishDate)`, note the whitespace.